styles/pattern-library.scss

```diff
@@ -217,3 +219,81 @@ body {
     background-color: var.$color-grey-1;
   }
 }
+
+$-library-vertical-spacing: 7;
+.LibraryPage {
```

Comment:
I wonder if `.LibraryPage` should belong to `styles/component/Library.css`

Reply:
I think it's worthwhile to consider some SASS module re-org after we get these new components used broadly and we're ready to remove the older support...I'll keep it in mind. I do want to keep pattern-library-related styling clearly separated from shared-component styling, however.